account_claims.go

```diff
@@ -129,6 +130,20 @@ func DecodeAccountClaims(token string) (*AccountClaims, error) {
 	if err := Decode(token, &v); err != nil {
 		return nil, err
 	}
+	// After decoding validation is complete, migrate imports
+	for _, i := range v.Imports {
+		if i.Subject != "" || i.To != "" {
+			if i.RemoteSubject != "" || i.LocalSubject != "" {
+				return nil, fmt.Errorf("import [%s] uses subject/to and remote/local - only remote/local or subject/to allowed for reading", i.Name)
+			}
+			i.RemoteSubject = i.Subject
```

Comment:
the meaning of subject and to depends on the type of the import. We need to do the flipflop here so that we don't have to do it anywhere else.